package.json

```diff
@@ -57,6 +59,7 @@
     "espree": "7.3.1",
     "esprima": "4.0.1",
     "fluent-syntax": "0.13.0",
+    "fsevents": "2.3.2",
```

Comment:
We don't need this dependency. NPM adds and removes it automatically.. and it's annoying. Could you remove it please?

Reply:
It actually appears in `optionalDependencies` as well. If I run `npm rm fsevents` it's removed from both. I sent a commit with this change